upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -196,6 +205,22 @@ func (_ *Subnet) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Subnet) error {
 		}
 	}
 
+	if e.IPv6CIDR != nil && strings.HasPrefix(aws.StringValue(e.IPv6CIDR), "/") {
```

Comment:
A comment would be helpful to indicate this handles the case where the VPC's IPv6CIDR was not known at the time `e.Find()` is run. 

And actually, `e.Find()` should `NewTryAgainLaterError` until it can get the association. Otherwise the code path is timing-dependent.

Reply:
This correct here, as it handles the case of `TargetDryRun`. We only care about the exact CIDR at render time, when we know that the VPC will eventually get and IPv6 CIDR.

Reply:
`TargetDryRun` will never call `RenderAWS()`, so that case will never happen.

Reply:
`Find()` has nothing to do with `VPC.IPv6CIDR`. It just needs to find the current state of the subnet.
If by the time it gets to `RenderAWS()` is still missing the `VPC.IPv6CIDR`, `NewTryAgainLaterError` waits for it.

Reply:
`Find()` is using `VPC.IPv6CIDR` to get the block to make allocations from.

`Find()` and `RenderAWS()` are using different rules for making the allocation and which one gets run depends on timing.